recipes/cppcheck/all/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(cmake_wrapper)
+
+message(STATUS "Conan CMake Wrapper")
+include("conanbuildinfo.cmake")
+conan_basic_setup(TARGETS)
+
+# Set include paths
+conan_global_flags()
+
+# Setup libraries to link
+set(Z3_LIBRARIES CONAN_PKG::z3)
+set(PCRE_LIBRARY CONAN_PKG::pcre)
```

Comment:
What are these for? I dont see where they are being used 🤔

Reply:
They are normally set in https://github.com/danmar/cppcheck/blob/488813d00fab8278a78d518cf0d3d6651c8f0caa/cmake/findDependencies.cmake and used in various places. Using the standard mechanism leaves out some other dependencies (in case of PCRE) or uses the wrong name (in case of z3).

Reply:
Ahhh thank you for the insight, I did not get to that file while searching the repo! Awesome work!